Core.ApplicationServices/Model/Notification/NotificationPermissions.cs

```diff
@@ -29,7 +28,7 @@ public NotificationPermissions(bool read, bool modify, bool delete, bool deactiv
                 return ReadOnly();
 
             var canBeModified = notification.IsActive && notification.AdviceType == AdviceType.Repeat;
-            var canBeDeactivated = canBeModified && !notification.AdviceSent.Any();
```

Comment:
I do not think this is enough - try the old UI on: https://staging.kitos.dk/#/system/usage/35402/advice
(Created for you to test)

The advis has been sent out and Deactivate can be clicked and the call to the backend returns with no errors. Editing is then prohibited.